content/docs/conditional-rendering.md

```diff
@@ -116,13 +120,16 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
+[**Probeer het op CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
 
-While declaring a variable and using an `if` statement is a fine way to conditionally render a component, sometimes you might want to use a shorter syntax. There are a few ways to inline conditions in JSX, explained below.
+Hoewel het verklaren van een variabele en het gebruiken van een `if` instructie een prima manier is om een component conditioneel te renderen, wil je soms een kortere syntax gebruiken.
+Er zijn een paar manieren om op dezelfde lijn condities in te voeren in JSX, zoals hieronder uitgelegd.
 
-### Inline If with Logical && Operator {#inline-if-with-logical--operator}
+### Inline If met Logische && Operator {#inline-if-with-logical--operator}
 
-You may [embed any expressions in JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) by wrapping them in curly braces. This includes the JavaScript logical `&&` operator. It can be handy for conditionally including an element:
+Je kan [alle expressies insluiten in JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) door ze in accolades in te sluiten.
+Dit omvat JavaScript's logische `&&`-operator.
+Het kan handig zijn om conditioneel een element mee te renderen:
```

Comment:
Not a proper sentence

Reply:
@Keraito It seems like a proper sentence to me. Could you point to the grammatical error(s) or suggest an alternative translation?

Reply:
In hindsight, my feedback was kind of useless and your sentence was not as "inproper" as I read at the time of review, apologies. I think that it's mainly hard to read as a sentence, even when I know what you're trying to say with it.

"Het kan" is too associated with the generic meaning representing "It is possible", which makes the current sentence really weird. I would suggest to actively reference the previous sentence with "Deze" or using "Hiermee" in the sentence.

If you want to stay close to the original content, I will suggest something like `Hiermee kan het handig zijn om een element conditioneel te renderen` or `Deze kan handig zijn om een element conditioneel mee te renderen`. Or if you want to be slightly more explicit: `Het kan handig zijn om deze te gebruiken voor het conditioneel renderen van een element`.

Reply:
I would even make the second sentence a subordinate clause (bijzin) of the first sentence:

```Hieronder valt ook de logische `&&`-operator uit JavaScript, waarmee je handig een element conditioneel kunt renderen.```

I think that we are used to having longer sentences in Dutch than in English (but I'm not certain it's true).

Reply:
Adapted to the following for now:
`
Hieronder valt ook de logische `&&`-operator uit JavaScript, waarmee je gemakkelijk een element conditioneel kunt renderen.
`